MinVer.Lib/GitCommand.cs

```diff
@@ -31,9 +33,9 @@ public static bool TryRun(string args, string workingDirectory, ILogger log, out
                 {
                     process.Start();
                 }
-                catch (Win32Exception ex)
+                catch (IOException ex)
                 {
-                    throw new Exception("Failed to run Git. Git may not be installed on the system.", ex);
+                    throw new Exception("The command \"git\" is inaccessible. Verify Git is installed and in the user PATH.", ex);

```

Comment:
How about we boil it down even further?

```suggestion
                    throw new Exception("\"git\" is not present in PATH.", ex);
```